schema-common.graphql

```diff
@@ -535,7 +535,7 @@ type Swap implements Event @entity {
   tokenIn: Token!
 
   " Amount of token deposited into pool in native units "
-  amountIn: BigDecimal!
+  amountIn: BigInt!
```

Comment:
may I ask why BigInt over BigDecimal? Is it because this is the amount in native units, for example, wei?

Reply:
yeah, it's documented here: https://github.com/messari/subgraphs/pull/10